index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://use.fontawesome.com/bba5c6ae87.js"></script>
+    <script src="https://use.fontawesome.com/83ac61cd0f.js"></script>
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header class="wrapper flex-nav">
+      <a class="logo item" href="#">Chidiebere</a>
+      <nav class="menu-icon toggle-icon item">
+        <i class="fa fa-bars" aria-hidden="true"></i>
+      </nav>
+    </header>
+
+    <section class="item-1">
+      <div class="headline">
+        <h1 class="intro name">Hey There. I'm Chidiebere Ojingwa</h1>
+        <h2 class="intro profession">I'am a Software Developer</h2>
+        <p class="intro task">
+          I can help you build a product, feature or <br />
+          website look through some of my work <br />and experience! If you like
+          what you see <br />
+          and have a project you need, <br />
+          don't hesitate to contact me
+        </p>
+
+        <ul class="social">
+          <li><a href="#" class="fa fa-github"></a></li>
+          <li><a href="#" class="fa fa-linkedin"></a></li>
+          <li><a href="#" class="fa fa-angellist"></a></li>
+          <li><a href="#" class="fa fa-twitter"></a></li>
+          <li><a href="#" class="fa fa-medium"></a></li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="item-2">
+      <!-- <div class="recentWork"> -->
```

Comment:
### OPTIONAL 

Don't forget before asking for a review, check that you removed every comment in your code. This will maintain your code cleaner and more organized 🙌